flow identifiers are path signals (#101)
intarchboard/draft-path-signals-collaboration: draft-iab-path-signals-collaboration.md

other parties to perform the task for which collaboration is desired,
and no more. This applies to information sent by an
application about itself, information sent about users, or information
sent by the network and it also includes information

> that can be used to associate packets to a flow or user context. 

Reviewer: Hmm... the original suggestion is quite lengthy sentence. But this new proposal seems to cut the part about "information sent by the network" which should stand alone, and only talks about information sent by network/endpoints that is used for managing flows...

> that can be used to associate packets to a flow or user context. 

Reviewer: Maybe: 

 This applies to information sent by an
 application about itself, information sent about users, or information
 sent by the network. This also applies to any information related to flow identification.

> sent by the network and it also includes information
> that can be used to associate packets to a flow or user context. 

Author: ```suggestion
sent by the network. This also applies to any information related to flow identification.
```

> that can be used to associate packets to a flow or user context. 

Reviewer 2: That works for me

This document recommends that signalling mechanisms that send information
are built to specifically support sending the necessary, minimal set of information (see {{minimize-info}})
and no more. As previsouly noted flow-identifying information are a path signal in itself,

> and as such provisioning of flow identifiers also requires protocol specific analysis.

Author: What do you not understand? It is mainly to answer the question in the issue that there is not one common way how to identify a flow.

> and as such provisioning of flow identifiers also requires protocol specific analysis.

Reviewer 2: "provisioning of flow identifiers also requires protocol specific analysis"... it seems a bit vague to just say that we need analysis. The previous sentence is about saying that we need to minimize data.

Shouldn't we be saying that "since flow identifiers are signals too, even they need to be minimized in how much they are reused", or something similar?

> and as such provisioning of flow identifiers also requires protocol specific analysis.

Author: why reused? The part of minimisation is probably covered in the earlier section. This one is more to say that it probably not always right to use the IP address (even if it often just there) but also design/select the identifiers based on the protocol used and function needed.

I agree my text might be vague but I also didn't want to add something length... or should we write more?
